tests/erp/export/test_export_integration.py

```diff
@@ -21,13 +20,6 @@ def test_csv_creation(db):
         export_schema_to_csv(dest_path, erps, EtalabMapper)
 
         assert Path(dest_path).exists() is True
-
-        resource = Resource(
-            "erp/export/static/exemple-valide.csv",
-            schema="erp/export/static/schema.json",
-        )
-        result = validate_resource(resource)
-        assert result.get("errors") == []
```

Comment:
Why is this test removed?

Reply:
I made a separated test for this at the bottom of the file. It wasn't the scope of that test at all, since the generated file wasn't tested against the ressource anyway.

Reply:
Oops indeed, sorry for the nois